### PR TITLE
deprecate scvi

### PR DESCRIPTION
**Pull request by mjayasur**

### CODE REVIEW

**Comment by adamgayoso** [comment]
Hi @ivirshup we were hoping to completely remove scvi from external. Users received notice about it's deprecation in the 1.7.X releases. I suppose this can wait until 1.8

**Comment by ivirshup** [comment]
Sounds good! I was wondering if that's what this is about.

We're trying to be good about semantic versioning, but it should be fine to merge this with the master branch once it's done (all 1.7.x stuff gets back ported to the 1.7.x branch, so we just won't backport this).

**Comment by adamgayoso** [comment]
> We're trying to be good about semantic versioning, but it should be fine to merge this with the master branch once it's done (all 1.7.x stuff gets back ported to the 1.7.x branch, so we just won't backport this).

Great! I'm not sure why it's failing though. Thanks!

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
       python -m pip install --upgrade pip
       pip install -r .pip-2033.txt
       pip install pytest-cov wheel
-      pip install .[dev,doc,test,louvain,leiden,magic,scvi,harmony,scrublet,scanorama]
+      pip install .[dev,doc,test,louvain,leiden,magic,harmony,scrublet,scanorama]
```
Comment (ivirshup) [inline .azure-pipelines.yml]:
```suggestion
      pip install .[dev,doc,test,louvain,leiden,magic,harmony,scrublet,scanorama,skmisc]
```

I think this should fix the azure builds

**Review by ivirshup — COMMENTED** [review]
It looks like the docs are failing since there are still links to `scvi` in the release notes. If you just change remove the `:func:` part from the references to this.

File: docs/release-notes/1.5.0.rst [hunk 2]
```diff
@@ -12,12 +12,12 @@ The `1.5.0` release adds a lot of new functionality, much of which takes advanta
 
 .. rubric:: New functionality
 
-- Many functions, like :func:`~scanpy.pp.neighbors` and :func:`~scanpy.tl.umap`, now store cell-by-cell graphs in :attr:`~anndata.AnnData.obsp` :pr:`1118` :smaller:`S Rybakov`
+- Many functions, like :func:`scanpy.pp.neighbors` and :func:`~scanpy.tl.umap`, now store cell-by-cell graphs in :attr:`~anndata.AnnData.obsp` :pr:`1118` :smaller:`S Rybakov`
```
Comment (ivirshup) [inline docs/release-notes/1.5.0.rst]: ?
```suggestion
- Many functions, like :func:`~scanpy.pp.neighbors` and :func:`~scanpy.tl.umap`, now store cell-by-cell graphs in :attr:`~anndata.AnnData.obsp` :pr:`1118` :smaller:`S Rybakov`
```

**Review by adamgayoso — APPROVED** [review]
LGTM

**Comment by ivirshup** [comment]
Thanks for fixing that up!

**Pull request merged**